### PR TITLE
Scale svg path to desired viewBox size

### PR DESCRIPTION
**Pull request by tanyaka**
…d viewBox size

See: https://www.npmjs.com/package/svg-path-parse

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "jest-serializer-vue": "2.0.2",
     "p-map": "^4.0.0",
     "prettier": "^2.8.3",
+    "svg-path-parse": "^1.1.3",
```
Comment (bromiesTM) [inline package.json:38]: It should be noted that this package has not been updated for eight years.
Alternative (2y no update https://www.npmjs.com/package/svgpath) also provides scale function

**Review by fracado — APPROVED** [review]
Although the library used seems to work fine, I found https://www.npmjs.com/package/svgpath might be a slightly better choice, as it is smaller, more recent and in my opinion a little bit easier to use.

So, instead of :
```
const pathDatas = pathParse(originalPath).normalize({ round: 2, transform: `translate(${translateX}, ${translateY}) scale(${scale})` })

result.icons[icon] = serializePath(pathDatas);
```
we would have:    
```
result.icons[icon] = svgpath(originalPath).round(2).translate(translateX, translateY).scale(scale).toString();
```

But as you prefer and I don't mind if we keep using `svg-path-parse`.

File: package.json (hunk 1 above)
Comment (fracado) [reply package.json:38]: Ha, I just recommended the same package... without having seen your comment. :sweat_smile:

**Comment by tanyaka** [comment]
Fine, we can use https://www.npmjs.com/package/svgpath instead!

**Pull request closed without merging**